README.md

```diff
@@ -273,6 +273,97 @@ manually).  One way to do this, on Linux, is to ensure that the
+
+In order to allow Visual Studio to configure the CMake project, the CMake configuration json will have to be editted to provide the [relevant options](#4-building-mlpack-from-source) needed to find all the dependencies. The options that you must provide to Visual Studio's CMake are:
+
+* `ARMADILLO_INCLUDE_DIR`
+* `ARMADILLO_LIBRARY`
+* `BOOST_ROOT`
+* `CEREAL_INCLUDE_DIR`
+* `BLAS_LIBRARIES`
+* `LAPACK_LIBRARIES`
+
+The CMake configuration json can be editted in Visual Studio by right clicking the root `CMakeLists.txt` in the project view, selecting **CMake settings for mlpack** and finally clicking on **edit JSON**. Adding new CMake option can be done by adding object fields with the following format to the `variables` array in the `CMakeSettings.json`:
+
+```json
+{
+    "name": "options_name_string",
+    "value": "options_value_string",
+    "type" : "{BOOL|FILEPATH|PATH|STRING}"
+}
+```
+Here is an example of a complete `CMakeSettings.json` needed to build mlpack on Visual Studio:
+
+<details>
+<summary>Click to see the example CMakeSettings.json</summary>
+
+```json
+{
+  "configurations": [
+    {
+      "name": "x64-Debug (default)",
+      "generator": "Ninja",
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${projectDir}\\out\\build\\${name}",
+      "installRoot": "${projectDir}\\out\\install\\${name}",
+      "cmakeCommandArgs": "",
+      "buildCommandArgs": "",
+      "ctestCommandArgs": "",
+      "variables": [
+        {
+          "name": "ARMADILLO_INCLUDE_DIR",
+          "value": "PATH/TO/CPP/DEPENDENCY/armadillo-10.1.2/include",
+          "type": "PATH"
+        },
+        {
+          "name": "ARMADILLO_LIBBRARY",
+          "value": "PATH/TO/CPP/DEPENDENCY/armadillo-10.1.2/lib/armadillo.lib",
+          "type": "PATH"
+        },
+        {
+          "name": "CEREAL_INCLUDE_DIR",
+          "value": "PATH/TO/CPP/DEPENDENCY/cereal-1.3.0/include",
+          "type": "PATH"
+        },
+        {
+          "name": "BUILD_ROOT",
+          "value": "PATH/TO/CPP/DEPENDENCY/boost_1_66_0",
+          "type": "PATH"
+        },
+        {
+          "name": "BOOST_INCLUDEDIR",
+          "value": "PATH/TO/CPP/DEPENDENCY/boost_1_66_0",
+          "type": "PATH"
+        },
+        {
+          "name": "BLAS_LIBRARIES",
+          "value": "PATH/TO/CPP/DEPENDENCY/OpenBLAS/lib/openblas.lib",
+          "type": "PATH"
+        },
+        {
+          "name": "LAPACK_LIBRARIES",
+          "value": "PATH/TO/CPP/DEPENDENCY/OpenBLAS/lib/openblas.lib",
+          "type": "PATH"
+        }
+      ]
+    }
+  ]
+}
+```
+</details>
+
+##### Blas and LAPACK on Windows
```

Comment:
```suggestion
##### BLAS and LAPACK on Windows
```
Technically BLAS is an acronym too, just like LAPACK. :)

Reply:
You are correct. I have replaced all the instances of `Blas` with `BLAS`